packages/itwin/grouping-mapping-widget/src/components/Groups/hooks/useFetchGroups.ts

```diff
@@ -2,24 +2,24 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-import type { IMappingsClient } from "@itwin/insights-client";
+import type { IGroupsClient } from "@itwin/insights-client";
+import { PreferReturn } from "@itwin/insights-client";
 import { useQuery } from "@tanstack/react-query";
 import type { GetAccessTokenFn } from "../../context/GroupingApiConfigContext";
 
 export const fetchGroups = async (
-  iModelId: string,
   mappingId: string,
   getAccessToken: GetAccessTokenFn,
-  mappingsClient: IMappingsClient
+  groupsClient: IGroupsClient
 )=> {
   const accessToken = await getAccessToken();
-  const groups = await mappingsClient.getGroups(accessToken, iModelId, mappingId);
-  return groups;
+  const groupsList = await groupsClient.getGroups(accessToken, mappingId, PreferReturn.Representation);
```

Comment:
Why do you need `Representation` here? `Minimal` should be enough since the widget does not do anything with group metadata, which is the only difference between Representation and Minimal.

Reply:
Changed `Representation` to `Minimal`

Reply:
Resolved